hack/docs/generate_samples.go

```diff
@@ -51,7 +50,7 @@ func UpdateComponentConfigTutorial() {
 }
 
 func UpdateCronjobTutorial() {
-	binaryPath := util.KubebuilderBinName
+	binaryPath := "/tmp/kubebuilder/bin/kubebuilder"
```

Comment:
Could we revert this one and still use the const?

Reply:
I think use the const 'util.KubebuilderBinName' couldn't work. This will not use the kubebuilder binary complied by the command. 🤔 See #3390

Reply:
It has been working so why it would stop to work within the changes of this PR?

Reply:
well, I think it has been working for only two days. The last commit changing it is : https://github.com/kubernetes-sigs/kubebuilder/commit/d0a1806c1d25c70777004271a4b773b490de3bcd

Reply:
This code is called when we run `make generate`.
The `make generate` is called when we do the CI checks and would NOT get merged if does not work. 

**See this code successfully executed to verify your PR: https://github.com/kubernetes-sigs/kubebuilder/actions/runs/4902477788/jobs/8754346010**

I can ensure that works either and you can properly very that by just running without these changes `make generate` locally within the master branch. 

Could you please revert these changes that have no relation to the scope of the PR?